definitions/procedures/content/switchover.rb

```diff
@@ -5,8 +5,7 @@ class Switchover < ForemanMaintain::Procedure
       for_feature :pulpcore
 
       confine do
-        # FIXME: remove this condition for the 6.10 upgrade scenario
-        !feature(:instance).downstream
+        !check_min_version(foreman_plugin_name('katello'), '4.0')
```

Comment:
I tried to depend on `katello-common` but removal of this package does not remove `tfm-rubygem-katello`, so instead used exact package name which has the rake task.

Reply:
i think we need to limit the migration procedure too, thats what actually failed in the associated bz

Reply:
nevermind!